integration_tests/chain.go

```diff
@@ -89,7 +89,7 @@ func (c *chain) configDir() string {
 	return fmt.Sprintf("%s/%s", c.dataDir, c.id)
 }
 
-func (c *chain) createAndInitValidators(count int) error {
+func (c *chain) createAndInitValidators(count int) error { // nolint:unused
```

Comment:
I left these functions in because the test was originally supposed to run with random keys. A more "correct" test would use random keys. To help debug, we added the ability to run with preset mnemonics, and from some previous convos it seemed like some people preferred to use the random ones.

No strong preference on if we keep them or not